### PR TITLE
chore(suite): update network symbol naming 6

### PR DESCRIPTION
**Pull request by adderpositive**
## Description
- update network symbol naming 

## Related 
Related https://github.com/trezor/trezor-suite/issues/14725

### CODE REVIEW

File: packages/suite/src/components/wallet/TransactionItem/InstantStakeBadge.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ export const InstantStakeBadge = memo(({ transaction, symbol }: InstantStakeBadg
         <Badge size="tiny">
             <Wrapper>
                 <Icon name="lightning" size={spacings.sm} />
-                <FormattedCryptoAmount value={amount} symbol={symbol} />
+                <FormattedCryptoAmount value={amount} symbolExtended={symbol} />
```
Comment (tomasklim) [inline packages/suite/src/components/wallet/TransactionItem/InstantStakeBadge.tsx]: i would leave it called `symbol`

Comment (adderpositive) [reply packages/suite/src/components/wallet/TransactionItem/InstantStakeBadge.tsx]: Ok

Comment (adderpositive) [reply packages/suite/src/components/wallet/TransactionItem/InstantStakeBadge.tsx]: also replaced `symbolExtended` in other components. It's cleaner to use only `symbol`. Anyway `symbolExtended` says nothing..

Comment (adderpositive) [reply packages/suite/src/components/wallet/TransactionItem/InstantStakeBadge.tsx]: updated in [77f55f2](https://github.com/trezor/trezor-suite/pull/15765/commits/77f55f2c4de1e67c4f0af9773e3911620371a2a6)

**Review by tomasklim — APPROVED** [review]

**Pull request merged**